symplyphysics/laws/thermodynamics/inner_energy_from_sum_heat_and_work.py

```diff
@@ -0,0 +1,39 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output)
+
+# Description
+# _______________________________________________________
+# LAW: Total energy of an isolated system is constant.
+# Energy can be transformed from one form to another, but
+# can neither be created nor destroyed.
+# Formula: dU = Q + A
+# Where:
+# dU is the change in internal energy of the system,
+# A is the work done by the system,
+# Q is the heat supplied to the system.
+# If the work is done by the system, and not by external
+# forces, then the formula is used:
+# dU=Q-A
```

Comment:
In description above A is work done by the system and it has a plus sign. Here it is still work done by the system, but with a minus sign. Please check what you are writing.